src/Deso.ts

```diff
@@ -1,16 +1,43 @@
 import axios, { AxiosInstance } from "axios";
 
+import { CustomError } from 'ts-custom-error'
+
 const DEFAULT_NODE_URL = "https://node.deso.org/api";
 
 let client: AxiosInstance | null;
 
+/**
+ * Represents an error returned by the API.
+ */
+class DeSoError extends CustomError {
+  public constructor(message?: string) {
+    super(message);
+  }
+}
+
 export class DesoClient {
   baseUrl: string;
 
   constructor({ baseUrl = DEFAULT_NODE_URL }: { baseUrl?: string }) {
     this.baseUrl = baseUrl;
   }
 
+  private async post(path: string, data: any) {
+    let result;
+
+    try {
+      result = await this.getClient().post(path, data);
```

Comment:
Any idea if this is the same error format for `get` requests too? If so maybe can add to all requests (maybe using an axios http interceptor or something which can apply to everything)